### PR TITLE
#567 docker image update

### PR DESCRIPTION
**Pull request by mi-we**
Image will be published to Docker hub after testing

### CODE REVIEW

File: docs/setup/Scenarioo-Viewer-Docker-Image.md [hunk 1]
```diff
@@ -12,15 +12,19 @@ To run the Scenarioo docker image it is necessary to install Docker. Visit the [
 
 You want to set up the scenarioo infrastructure and run the Scenarioo webapplication? It is a one-liner:
 
-1. Run `sudo docker run -it --rm --name scenarioo -p 8080:8080 -v $(pwd)/scenarioDocuExample:/doku scenarioo/webapp:2.2.0`  
+1. Run `sudo docker run -it --rm --name scenarioo -p 8080:8080 -v $(pwd)/scenarioDocuExample:/docu scenarioo/webapp:3.0.0`
```
Comment (bruderol) [inline docs/setup/Scenarioo-Viewer-Docker-Image.md]: make a better example than '$(pwd)/scenarioDocuExample'  - maybe use just /home/user/your-scenarioo-host-docu-directory-path

File: docs/setup/Scenarioo-Viewer-Docker-Image.md [hunk 2]
```diff
@@ -12,15 +12,19 @@ To run the Scenarioo docker image it is necessary to install Docker. Visit the [
...
 You want to set up the scenarioo infrastructure and run the Scenarioo webapplication? It is a one-liner:
 
-1. Run `sudo docker run -it --rm --name scenarioo -p 8080:8080 -v $(pwd)/scenarioDocuExample:/doku scenarioo/webapp:2.2.0`  
+1. Run `sudo docker run -it --rm --name scenarioo -p 8080:8080 -v $(pwd)/scenarioDocuExample:/docu scenarioo/webapp:3.0.0`  
 with following parameters:    
 `--name` - defines a name for the container (= running image)  
-`-p` - maps a port of your host OS to the container internal port.  
-`-v` - defines where the Sceanrioo-Docu-Directory is located (watch out: if you use $(pwd) you have to run the command in a specific directory)  
+`-p` - maps a port of your host OS to the container's internal port.  
+`-v` - maps the directory where your Scenarioo documentation data and config will be stored (watch out: if you use $(pwd) you have to run the command in a specific directory)
```
Comment (bruderol) [inline docs/setup/Scenarioo-Viewer-Docker-Image.md]: Remove `(watch out: if you use $(pwd) you have to run the command in a specific directory)  ` when we used a better example and explain that this directory is mapped to `docu` inside the image.

File: docs/setup/Scenarioo-Viewer-Docker-Image.md [hunk 3]
```diff
@@ -12,15 +12,19 @@ To run the Scenarioo docker image it is necessary to install Docker. Visit the [
...
-`-p` - maps a port of your host OS to the container internal port.  
-`-v` - defines where the Sceanrioo-Docu-Directory is located (watch out: if you use $(pwd) you have to run the command in a specific directory)  
+`-p` - maps a port of your host OS to the container's internal port.  
+`-v` - maps the directory where your Scenarioo documentation data and config will be stored (watch out: if you use $(pwd) you have to run the command in a specific directory)  
 
-2. Access [http://localhost:8080/scenarioo/](http://localhost:8080/scenarioo/) in your webbrowser to verify the running webclient.
+2. Access [http://localhost:8080/scenarioo/](http://localhost:8080/scenarioo/) in your webbrowser to verify the Viewer application is running.
 
-## More informations
+3. At this point, the Scenarioo configuration is only in memory. Go to Manage -> General Settings, add your application name and save the configuration.
```
Comment (bruderol) [inline docs/setup/Scenarioo-Viewer-Docker-Image.md]: Mention that config file can then be adjusted manually and is located in the mapped documentation directory. Maybe link to configuration docu?

File: docs/setup/Scenarioo-Viewer-Docker-Image.md [hunk 4]
```diff
@@ -12,15 +12,19 @@ To run the Scenarioo docker image it is necessary to install Docker. Visit the [
...
+
+4. Start adding your documentation data to the mapped directory and let the Viewer application import it.
```
Comment (bruderol) [inline docs/setup/Scenarioo-Viewer-Docker-Image.md]: Maybe link to "how to publish docu" documentation?

**Comment by bruderol** [comment]
@mi-we see my proposals for improvements. I will try to test soon.

**Comment by bruderol** [comment]
@sueess what do you think? looks good?

**Comment by danielsuter** [comment]
@bruderol @mi-we 
Maybe it would make sense to fix the graphics magick version? Otherwise the image is not reproducible. What do you think?

**Comment by bruderol** [comment]
@danielsuter not understand exactly what you mean ??

**Comment by danielsuter** [comment]
graphics magick is installed using apt-get install. Thus the version we get changes every time ubuntu updates it, which in turn could cause an incompatibility with our code. That results in people downloading our docker image and it's not working.

**Comment by mi-we** [comment]
@bruderol @danielsuter I updated the Dockerfile with a modified apt-get install command as suggested in this StackOverflow thread: https://askubuntu.com/questions/92019/how-to-install-specific-ubuntu-packages-with-exact-version#92021

This should lock the graphicsmagick version to 1.3.20 now (the lowest version in the repository of the underlying Debian Jessie distribution of the Tomcat docker image).

**Comment by bruderol** [comment]
@sibetester tested the new docker image together with me for us, it seems to be okay. 
I will try to finalize review of docu and then we could merge it.

**Comment by bruderol** [comment]
@mi-we I reviewed and tested it and committed some improvements (precising) to the documentation. 

Can you please review my changes and then if you agree:
Just publish the image to DockerHub and merge this PR.

Thanks a lot for all! 👍

**Comment by mi-we** [comment]
I uploaded the new image to Dockerhub, did a short test that it can be downloaded and started.

**Comment by bruderol** [comment]
Cool :-) Thanks ! 🔝 👍

**Pull request merged**